lib/ruleHelper.js

```diff
@@ -110,6 +111,12 @@ RuleHelper.prototype = {
      * @returns {boolean} Returns whether the Identifier is deemed safe.
      */
     isAllowedIdentifier(expression, escapeObject, details) {
+
+        // respect the custom config property `variableTracing`:
+        if (this.ruleChecks["variableTracing"] < 1) {
```

Comment:
@mozfreddyb this is more like a doubt/questions then a blocking issue: can we make it a boolean setting instead?

Reply:
Originally, the idea was that if we end up expanding this feature, we might want to call this "level 2" and make it opt-in as well.
However, coming to think of it, I don't feel overly enthusiastic maintaining too many options and variations thereof :) I'll turn it into a bool.

Reply:
Can this be changed to `if (!this.ruleChecks["variableTracing"]) { return false; }`?